CMakeLists.txt

```diff
@@ -105,6 +105,11 @@ else()
   set(CMAKE_CXX_EXTENSIONS OFF)
 endif()
 
+if (TILEDB_CCACHE)
+  include(FindCcache)
+  set(CMAKE_CXX_COMPILER_LAUNCHER ${CCACHE})
```

Comment:
I don't know anything about TileDB, but it looks like you have some C sources under `examples/`, so you probably want to set `CMAKE_C_COMPILER_LAUNCHER` as well.

Reply:
Done, thanks!